integration_tests/tests/match/match.cy.ts

```diff
@@ -25,17 +25,39 @@ context('Placement Requests', () => {
     cy.task('stubBedSearch', { bedSearchResults })
     const person = personFactory.build()
     cy.task('stubFindPerson', { person })
-    const placementRequests = placementRequestFactory.buildList(1, { person })
-    cy.task('stubPlacementRequests', placementRequests)
+    const activePlacementRequest = placementRequestFactory.build({ person, status: 'not_matched' })
```

Comment:
Is there a reason why we can only have one active placement request? 
I think it would be better to `placementRequestFactory.buildList(1, { person, status: 'not_matched' })` 
so we don't have to treat it differently later in the test (EG line 33, 42)